BookingSpecBindings/TestBase/ExpectedSearch.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using TechTalk.SpecFlow;
+
+namespace BookingSpecBindings.TestBase
+{
+	public class ExpectedSearch
+	{
+		private string Price = "*";
+		private string Stars = "*";
+		private string Rating ="*";
+		private string FreeWIFI = "*";
+
+		public ExpectedSearch(Table table)
+		{
+			foreach (var row in table.Rows)
```

Comment:
Looks like you will store and compare only last row. For both actual and expected searches you need to have collections. Here you need to put row in the constructor and in the step you need to create collection of ExpectedSearch elements